### DISCUSSION TITLE
1.16.5 minecolonie

### DISCUSSION THREAD

**Discussion by LostSagaX** [ldtteam/minecolonies#8401] [Q&A]
Barrack tower not spawning properly and tried to fix it by manually placing a new one but does not work what do i do
![2022-05-24_17 51 56](https://user-images.githubusercontent.com/106179860/170138586-fdf6272b-d2cc-4b09-a66f-a166c6d0939f.png)
.

**Comment by uecasm** [comment]
It's best to ask support questions on the Discord.

But the jungle style barracks does not work in 1.16.  You need to build a different style instead, or play on 1.18.

**Comment by LostSagaX** [comment]
oh ok
